src/main/java/buildcraftAdditions/blocks/BlockKineticEnergyBufferTier1.java

```diff
@@ -57,7 +57,7 @@ public void registerBlockIcons(IIconRegister register) {
 
 	@Override
 	public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer player, int side, float hitX, float hitY, float hitZ) {
-		if (world.getTileEntity(x, y, z) instanceof TileKineticEnergyBufferBase)
+		if (!world.isRemote)
```

Comment:
the instanceof check is needed, never just asume the TE is the correct one, something might get messed up somewhere. mostly caused by other mods moving blocks but it could heapen